### PR TITLE
User Docs Update

### LINKED ISSUE #72: Update the instructions for conf.py in theme docs
We should make these instructions say either delete or modify the on_rtd instructions for the new theme: https://github.com/jupyterhub/jupyterhub/blob/master/docs/source/conf.py#L172

```
on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
if not on_rtd:
    import jupyter_alabaster_theme
    html_theme = 'jupyter_alabaster_theme'
    html_theme_path = [jupyter_alabaster_theme.get_path()]
else:
    # readthedocs.org uses their theme by default, so no need to specify it
    # build rest-api, since RTD doesn't run make
    from subprocess import check_call as sh
    sh(['make', 'rest-api'], cwd=docs)
```

### PR DESCRIPTION
**Pull request by charnpreetsingh**
Closes #72 

@willingc Please let me know what you think or a better way to word the instructions

### CODE REVIEW

File: docs/source/user.rst [hunk 1]
```diff
@@ -32,15 +32,38 @@ Edit your ``conf.py`` file:
     ]
 
 3. At the bottom of ``conf.py``, if the following block of code is present,
-   remove it:
+   remove or modify it, otherwise ``sphinx_rtd_theme`` or the default readthedocs
```
Comment (willingc) [inline docs/source/user.rst]: edit the code block to select the desired theme to use for any local builds (such as during development):

*block to look for here*


If you are building documentation locally, you may specify a desired theme (jupyter_alabaster_theme) or the sphinx_rtd_theme:

*jupyter_alabaster theme* - use #72 code

*sphinx_rtd_theme - show example below.

**Review by willingc — CHANGES_REQUESTED** [review]
Double check that building without a theme specified in the `if not on_rtd` does in fact build with the new alabaster theme. It was not doing that when I worked with the theme and jupyterhub last week.

**Comment by charnpreetsingh** [comment]
@willingc  I verified that `jupyter_alabaster_theme`  does not need to be imported (via a clean conda environment) if developing locally. Setting the html_theme and adding it to extensions is all that's needed to use the theme

**Review by willingc — APPROVED** [review]
Thanks @charnpreetsingh. Looking good :cake:

**Pull request merged**